### PR TITLE
Downgrade back to Tokio 0.2

### PR DESCRIPTION
**Pull request by romac**
Closes: #415 

______

For contributor use:

- [ ] Updated the __Unreleased__ section of [CHANGELOG.md](https://github.com/informalsystems/ibc-rs/blob/master/CHANGELOG.md) with the issue.
- [ ] If applicable: Unit tests written, added test to CI.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Updated relevant documentation (`docs/`) and code comments.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -12,16 +12,7 @@ exclude = [
...
-
-bytes = { git = "https://github.com/tokio-rs/bytes", tag = "v0.6.0" }
-
-tonic = { git = "https://github.com/romac/tonic", branch = "romac/hyper-0.14-dev" }
-
-prost        = { git = "https://github.com/danburkert/prost", branch = "master" }
-prost-types  = { git = "https://github.com/danburkert/prost", branch = "master" }
-prost-build  = { git = "https://github.com/danburkert/prost", branch = "master" }
-prost-derive = { git = "https://github.com/danburkert/prost", branch = "master" }
+tendermint              = { git = "https://github.com/informalsystems/tendermint-rs", branch = "master" }
```
Comment (adizere) [inline Cargo.toml:15]: Maybe we don't need master, and can stick to RC3 ? https://github.com/informalsystems/tendermint-rs/commit/91bc1f6170528c82fb1aacc370e9d75d7e571bd2

**Review by ancazamfir — APPROVED** [review]
Looks good! I tested all CLIs and working fine.

File: Cargo.toml (hunk 1 above)
Comment (ancazamfir) [reply Cargo.toml:15]: yeh, that would be nice

Comment (romac) [reply Cargo.toml]: Done

**Pull request merged**